common/src/main/resources/framework.properties

```diff
@@ -0,0 +1 @@
+version=${version}
```

Comment:
Totally minor, but add newlines at the end of the files (helps with greping etc..)

Reply:
Done